accessman.go

```diff
@@ -611,8 +611,7 @@ func (a *accessMan) addPeerAccess(remotePub *btcec.PublicKey,
 
 // removePeerAccess removes the peer's access from the maps. This should be
 // called when the peer has been disconnected.
-func (a *accessMan) removePeerAccess(peerPubKey string) {
-	ctx := btclog.WithCtx(context.TODO(), "peer", peerPubKey)
+func (a *accessMan) removePeerAccess(ctx context.Context, peerPubKey string) {
```

Comment:
I propose to change other methods for consistency. Currently other methods generate a ctx from context.Background() and do pubkey->str conversion inside:
  * assignPeerPerms
  * newPendingOpenChan
  * newPendingCloseChan
  * newOpenChan
  * checkAcceptIncomingConn
  * addPeerAccess

Reply:
yeah def - but let's do that in a follow up? This PR is just for fixing the logging bug